Build/loc/TranslationsImport.yml

```diff
@@ -0,0 +1,33 @@
+# ==================================================================================
+# Pipeline for VsCodeExtension-Localization build definition
+# Gets translated files from VSCodeExt and creates a PR for vscode-cpptools
+#
+# Triggered only by daily schedule
+# ==================================================================================
+
+trigger: none
+pr: none
+schedules:
+- cron: "0 7 * * *"
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+- checkout: self
+  path: s/vscode-cpptools
+
+- task: CmdLine@2
+  inputs:
+    script: 'git clone $(LocRepoUri) ..\VSCodeExt'
+    workingDirectory: '$(Build.SourcesDirectory)'
+
+- task: CmdLine@2
+  inputs:
+    script: 'cd Extension && yarn install'
+    workingDirectory: '$(Build.SourcesDirectory)'
+
+- task: CmdLine@2
+  inputs:
+    script: 'cd Extension && node ./translations_auto_pr.js microsoft vscode-cpptools cpptools csigs $(csigsPat) "cpptools service account" cpptools@microsoft.com $(Build.SourcesDirectory)/../VSCodeExt'
```

Comment:
Can you remove the direct reference to the internal account as well?  We are retiring this account anyway.

Reply:
I need to set the email and user name of the git commit to something.  That something will be displayed in the (public) git log, so there is no use in hiding it here.  Do you have a suggestion on what that user name and email should be?

Reply:
Previously, we were picking up the default configured git user name and email from the build machine.  i.e.:
DevDiv Build Lab - Dev14 <dlab14@microsoft.com>

But, when using YML, the build machine is not configured with a user name or email, and cannot commit unless I set them.

Provided that the PR is squashed when committed, that info will be replaced with the name and email of associated with the PR (csigs) :

csigs <csigs@users.noreply.github.com>

I suppose I could use that same account information for the commit.